### PR TITLE
Update help docs to include oras transport

### LINKED ISSUE #3767: No help for Singularity ORAS
## Version of Singularity:

3.3.0-rc1

### Expected behavior

A help doc for `singularity oras --help`

### PR DESCRIPTION
**Pull request by ikaneshiro**
Signed-off-by: Ian Kaneshiro <iankane@umich.edu>

Fixes: #3767

### CODE REVIEW

File: docs/content.go [hunk 1]
```diff
@@ -565,20 +571,36 @@ Enterprise Performance Computing (EPC)`
...
+  The 'push' command allows you to upload a SIF container to a given
+  URI.  Supported URIs include:
+
+  library:
+      library://user/collection/container[:tag]
+
+  oras:
+      oras://registry/namespace/repo:tag
+
+
```
Comment (WestleyK) [inline docs/content.go:594]: Extra newline here? Is this supposed to be here?

**Review by WestleyK — APPROVED** [review]
Looks good to me, just one comment about a "extra" newline :+1:

File: docs/content.go (hunk 1 above)
Comment (ikaneshiro) [reply docs/content.go:594]: That was intentional, I wanted some space before the note, but if you think it would look better without I can remove it

Comment (WestleyK) [reply docs/content.go:594]: Imo, its looks better without the extra newline, but I'm cool either way.

File: docs/content.go [hunk 2]
```diff
@@ -60,7 +60,8 @@ Enterprise Performance Computing (EPC)`
 
       library://  an image library (default https://cloud.sylabs.io/library)
       docker://   a Docker registry (default Docker Hub)
-      shub://     a Singularity registry (default Singularity Hub)`
+      shub://     a Singularity registry (default Singularity Hub)
+      oras://     a supporting OCI registry (no default)`
```
Comment (jscook2345) [inline docs/content.go]: Remove `(no default)`

File: docs/content.go [hunk 3]
```diff
@@ -556,7 +559,10 @@ Enterprise Performance Computing (EPC)`
       docker://user/image:tag
     
   shub: Pull an image from Singularity Hub to CWD
-      shub://user/image:tag`
+      shub://user/image:tag
+
+  oras: Pull an SIF image from a supporting OCI registry
```
Comment (jscook2345) [inline docs/content.go]: Change `Pull an SIF` to `Pull a SIF`.

File: docs/content.go [hunk 4]
```diff
@@ -556,7 +559,10 @@ Enterprise Performance Computing (EPC)`
...
+      oras://registry/namespace/repo:tag`
```
Comment (jscook2345) [inline docs/content.go]: Should `repo` be `image` instead?

File: docs/content.go [hunk 5]
```diff
@@ -565,20 +571,36 @@ Enterprise Performance Computing (EPC)`
   $ singularity pull tensorflow.sif docker://tensorflow/tensorflow:latest
 
   From Shub
-  $ singularity pull singularity-images.sif shub://vsoch/singularity-images`
+  $ singularity pull singularity-images.sif shub://vsoch/singularity-images
+
+  From supporting OCI registry (e.g. Azure Container Registry)
+  $ singularity pull image.sif oras://<username>.azurecr.io/namespace/repo:tag`
```
Comment (jscook2345) [inline docs/content.go]: #1. Can we put a real example here?
#2. `repo` vs `image` again.

File: docs/content.go [hunk 6]
```diff
@@ -565,20 +571,36 @@ Enterprise Performance Computing (EPC)`
...
+  NOTE: It's always good practice to sign your containers before
+  pushing them to the library. An auth token is required to push to the library,
+  so you may need to configure it first with 'singularity remote'.`
 	PushExample string = `
-  $ singularity push /home/user/my.sif library://user/collection/my.sif:latest`
+  To Library
+  $ singularity push /home/user/my.sif library://user/collection/my.sif:latest
+
+  To supported OCI registry
+  $ singularity push /home/user/my.sif oras://registry/namespace/repo:tag`
```
Comment (jscook2345) [inline docs/content.go]: `repo` vs `image` again

**Review by jscook2345 — CHANGES_REQUESTED** [review]
I put some comments in for review. Ping me when you need a re-review! Thanks!

File: docs/content.go (hunk 4 above)
Comment (ikaneshiro) [reply docs/content.go]: The nomenclature I've seen more is to call that a repo and then the image is the repo plus tag.

i.e. https://hub.docker.com/_/golang, the repo is `golang` and then you specify which image you want based on the tag

File: docs/content.go (hunk 5 above)
Comment (ikaneshiro) [reply docs/content.go]: 1. If we want a public acr registry for people to pull from to try out, then we would need to set that up ourselves. I tested using a trial and am not planning on making the registry public or extending it once it expires.

2. See above.

**Comment by ikaneshiro** [comment]
@jscook2345 ping!

**Review by jscook2345 — APPROVED** [review]
LGTM

**Pull request merged**